spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -133,6 +133,7 @@ def assert_specific_vm
         :description      => "PVM Instance",
         :vendor           => "ibm_power_vs",
         :power_state      => "on",
+        :placement_group  => nil,
```

Comment:
Would be nice to also include specs for a VM with an associated placement group.

Reply:
Done in cc7f886d